src/libraries/Microsoft.PowerFx.Interpreter/UDF/CheckWrapper.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Interpreter.UDF
+{
+    internal class CheckWrapper
```

Comment:
Could we get a comment describing what this class is for?

Reply:
done